.infrastructure/5_iam.tf

```diff
@@ -0,0 +1,86 @@
+resource "aws_iam_role" "ecs-task-execution-role" {
+  name = "ecs_task_execution_role"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+
+  tags = {
+    tag-key = "tag-value"
+  }
```

Comment:
I think this is useless :)

Reply:
Correct, I'll remove it from the code with the next commit